diff --git a/libero/libero/benchmark/__init__.py b/libero/libero/benchmark/__init__.py
index 2b833d0..11c8812 100644
--- a/libero/libero/benchmark/__init__.py
+++ b/libero/libero/benchmark/__init__.py
@@ -161,7 +161,10 @@ class Benchmark(abc.ABC):
             self.tasks[i].problem_folder,
             self.tasks[i].init_states_file,
         )
-        init_states = torch.load(init_states_path)
+        try:
+            init_states = torch.load(init_states_path, weights_only=False)
+        except:
+            init_states = torch.load(init_states_path)
         return init_states
 
     def set_task_embs(self, task_embs):
diff --git a/libero/libero/envs/venv.py b/libero/libero/envs/venv.py
index ed1fd40..28bd495 100644
--- a/libero/libero/envs/venv.py
+++ b/libero/libero/envs/venv.py
@@ -1,6 +1,6 @@
 import cloudpickle
 import ctypes
-import gym
+import gymnasium as gym
 import numpy as np
 import numpy as np
 import warnings
