diff --git a/robosuite/macros.py b/robosuite/macros.py
index c49c84b6..9172714f 100644
--- a/robosuite/macros.py
+++ b/robosuite/macros.py
@@ -47,9 +47,4 @@ FILE_LOGGING_LEVEL = None
 try:
     from robosuite.macros_private import *
 except ImportError:
-    import robosuite
-    from robosuite.utils.log_utils import ROBOSUITE_DEFAULT_LOGGER
-
-    ROBOSUITE_DEFAULT_LOGGER.warn("No private macro file found!")
-    ROBOSUITE_DEFAULT_LOGGER.warn("It is recommended to use a private macro file")
-    ROBOSUITE_DEFAULT_LOGGER.warn("To setup, run: python {}/scripts/setup_macros.py".format(robosuite.__path__[0]))
+    pass
